Lib/asyncio/tasks.py

```diff
@@ -19,6 +21,8 @@
 from . import events
 from . import futures
 from .coroutines import coroutine
+from .base_tasks import (all_tasks, current_task, _register_task,
```

Comment:
Please import the module, not functions from it.

Reply:
In this case `base_tasks` should have `__all__ = (`all_tasks`, `current_task`)` to export names into `asyncio` namespace. `_register_task` cannot be exposed by `base_tasks`, see comment below.